### DISCUSSION TITLE
Платный разбан

### DISCUSSION THREAD

**Discussion by quillincn** [octothorp-team/public-dobrograd#819] [Предложения]
# 🚪 Предисловие
Перед прочтением нужно сделать глубокий вдох и выдох, а после постараться оградиться от предыдущего опыта продажи разбана на других серверах и посмотреть на предложенную идею, как на возможность улучшить Доброград, но никак не наоборот.

# 🧪 Идея
Здесь всё просто – продавать разбан, но на определённых условиях и для определённой цели. Ещё раз нужно выделить, эта идея не о том, как делают большинство серверов, а о возможности поддержать проект и администрацию в первую очередь.

## 🔐 Условия покупки разбана
1. Бан не должен быть перманентным и превышать определённое время, например, если срок не достигает 2-ух недель.
2. При выдаче наказания, администратор в праве устанавливать возможность покупки разбана, все ситуации уникальны и поэтому оставлять право выбора за проектом, кому доступна эта плюшка а кому нет, будет разумно.
3. Лимит на покупку по времени или по количеству, например, разбан можно будет купить раз в 1-2 месяца или в общей сложности всего до 3-ех раз.

Список может быть дополнен, это примерные условия.

## 🪝 Мотивация против нарушений
При покупке разбана игрок будет ценить не просто потраченное время (жалоба или окончание срока), но и деньги. Теперь от нарушений его удерживают его личные средства, те же 250-350 рублей, которые уже куда серьёзнее заставляют задуматься о последствиях нарушений правил и порчи игры.

## 🤞🏼 Исключения
При словосочетании "платный разбан" сразу возникает мысль о людях, для которых не является затратным снимать с себя наказание за деньги. На самом деле лишних денег не бывает, а таких игроков крайне мало и даже если они есть, их будет ограничивать условия: решение администратора о возможности покупки как таковой, лимит на покупку.

##  🤵 Репутация
Вне зависимости от названия "услуги" или "плюшки", у людей в голове плотно засела параллель о плохом качестве проекта, если там можно снять с себя наказание за деньги. Поэтому, при реализации можно давать пояснение о мотивации и ограничениях и куда пойдут эти средства. Чтобы до каждого дошло, что это не о жадности, а о возможностях и все максимально толково, как и всегда.

## 🎯 Цель
Объективную оценку решению проблем могут дать только некоторые игроки, ситуация с администрацией стала в разы лучше, но идеалу нет предела. На это по большей части и могут быть направлены средства с платных разбанов, в карман толковым и заинтересованным админам. Вознаграждение за старания лишний раз будут давать мотивацию вести проект дальше.

## 👀 Спрос
Любой вход в голосовой канал Discord – история о том, как кто-то попал в бан. В той же базе данных можно просмотреть количество не перманентных банов или провести опрос через форму, о том, как люди отнесутся к такому формату доната и заинтересованы ли они в его покупке.

## 📐 Реализация
Это может быть отдельная вкладка на странице пополнения, туда указывается SteamID, после происходит проверка возможности покупки и возможно какое-нибудь соглашение с правилами, чтобы еще раз подчеркнуть сам смысл траты, а именно возвращение на сервер для здравой игры без помехи окружающим.

## 🧶 Заключение
Если после прочтения всё еще остались какие-то сомнения или осадок, просьба еще раз задуматься над возможностью, которую принесет эта услуга. Это не окончательная мысль, всё может меняться и подстраиваться, тут только пример того, как это можно было бы реализовать, а условия, цели и прочее можно продумывать куда детальнее. 

В любом случае все это только в интересах игроков, поддержать администрацию рассмотрев такую возможность под другим углом или оставить всё как есть.

**Comment by pr0pk1** [comment]
Квиллин все пояснил, базар. Реакции негативные, а я за.

**Comment by KAGIRA-777** [comment]
- Вы же понимайте, что сумма в 250-300 рублей не особо велика и ее может потянуть любой школьник, да и что ограничение в 1-2 месяца на покупку разбана радужную картину не строит ?
- Насколько сильно администрация будет объективна выдавая игрокам баны без возможности покупки разбана ?
- Может ли быть в теории какой-нибудь приказ для администрации чтоб они выдавали баны людям и оставляли возможность покупать разбаны, например когда за месяц не набралась определенная сумма денег ?
- Вы же понимайте, что с вводом данной возможности все олды, да и не только олды, доброграда начнут активно вас хейтить ? Олды и до этого не особо любили новые реалия доброграда, а сейчас у них появился еще один повод.
- И последний вопрос - Вы и правда не нашли иной способ дохода, который не был настолько презираемым для всех рп-игроков ? Это же очень сильно повлияет на репутацию проекта.

**Reply by quillincn** [reply]
Без негатива, но в предложении чётко сказано, что это только пример. Я не должен и не могу продумать все аспекты. Над любой задумкой в рамках большого проекта должно работать много людей, которые проработают всё до мелочей. Идея на то и называется идеей, её нужно развивать.

1. Так ведь это хорошо, раз такая сумма посильна большому количеству игроков. Тем более это совсем нефинальная цифра, она взята с потолка, можно ведь установить совсем другую цену. Тем более, если не все, то почти все, опираясь на твоё предположение, имеют свободные 250-350 рублей, но при этом они не заинтересованы в покупке других плюшек, тогда это могло бы стать достаточно хорошим вариантом.
2. Не знаю, возможно этот выбор стоит оставить за старшей администрацией или отладить систему требований к игроку, которая сама будет определять возможность покупки.
3. Можно зайти в список плюшек и это станет ответом. У Доброграда нет цели менять баланс игры в угоду прибыли. А если это и случится в теории, что кто-то там потребует чаще выдавать баны, такой администратор надолго не задержится.
4. Олдов на Доброграде практически нет, по крайней мере в моём понимании слова. Людям в принципе не нравятся любые перемены, любые нововведения популярных сервисов встречают негатив: вспомнить тот же редизайн ВК, отключение дизлайков на YT, как все не хотели переходить на Windows 10. Всё принималось и принимается в штыки, а после преживается и отлично работает. Всегда надо что-то пробовать.
5. А что плохого в платном разбане, если всё можно организовать цивильно? Все остаются в плюсе, игроки, решившие не подавать заявку, экономят время разбора жалобы и мотивируются не нарушать из-за потраченных средств, а админы получают поддержку.

Не нужно смотреть на какие-то сервера, где народ плотно качают на бабки, где никто не заинтересован в качестве аудитории и самой игры. В этой ситуации никто не останется в проигрыше, те кто хотят вредить – пойдут обходить и споткнуться, а кто заинтересован, решатся на покупку. Никто не заплатит значимые деньги ради нарушений, а если и будут не разумные люди, их можно поблагодарить за поддержку и мирно отправить обратно в бан.

**Reply by KAGIRA-777** [reply]
@quillincn 
1. Ну смотри-ка почему я не считаю, что большая доступность это хорошо.  Как минимум это понизит значимость такого явления как баны, это приведет к тому, что люди не будут его бояться. Я помню постоянно боялся банов и старался играть исключительно по правилам, в итоге я получил только 1 бан за два года игры. Да, я понимаю, что страх как элемент управления не особо гуманен, но все же он действенен.
2. -
3. Знаешь, это просто мне напоминает ситуацию с Ассасином и юбисофт, они понемногу вводили мелкие донаты и прикольчики разные, а потом бац и все, чтоб получить контент нужно отдать всю свою зарплату. Конечно сравнивать 3A игру и сервер в гаррис моде это конечно немного странно, но сходства некие прослеживаются.
4. Да, я согласен, что некоторые обновления принимаются в штыки, а потом к ним привыкают, но сейчас до сих пор есть люди которые сидят на виндов 7, сидят с расширением дизлайков на ютубе и старым дизайном в вк. Но дело в том, что некоторые старые игроки и не приняли новые обновления, они ушли. Вслед за ними могут уйти и другие старички, которые терпели, терпели не поймут данный концепт и посчитают, что сервер скатился. Если вы готовы к кому-либо оттоку игроков - ваше решение.
5. Ну если у вас получиться все это организовать гуманно - это будет хорошо.

Я здесь чет критикую, но все же лучше помочь мелким дополнением. Вы можете взымать деньги как залог за разбан, эти деньги у вас держится до окончания срока бана или еще дольше, а потом вы возвращайте 100%, либо 50% от суммы обратно в карман игроку. Но если он неподобающе себя ведет в течении этого срока - деньги вы не возвращайте, ну и еще можно бан вдогонку дать. Почему этот концепт лучше банального разбана ? Все просто, у игрока будет мотивация не нарушать поставленные перед ним условия. Опять же страх как элемент управления, но все же он действенен. Значимость бана не уменьшится при этом концепте. Вы можете принять данный концепт, вы вряд ли очень много денег получите с данного нововведения, но для вас деньги не главно, с ваших же слов. А этот концепт поможет поддерживать порядок на сервере.

**Reply by quillincn** [reply]
1. Ну так ёлки-маталки, йошки-матрешки, ёк-макарек, В самом предложении уже описано, выражаясь прямо, страх потери возможности играть на сервере меняется на страх потерять деньги. Сумма не окончательная, даже если это 350 рублей, какой смысл выбрасывать их на воздух? Игрок тратит эти деньги для продолжения игры, а не чтобы нарушить и опять попасть в бан. На мой взгляд деньги мотивируют куда лучше в этом случае. И опять же, раз все могут позволить себе такую трату, но далеко не все покупают того же Добродея, эта форма доната могла бы привлечь хорошие средства для поддержки администрации.
3. Примеры как аргумент в этом случае возможно не самое удачное решение, но сама суть в том, чтобы попробовать что-то новое. Не обязательно браться за каждое предложение, раз оно "не стандартное" и "рисковое", просто не стоит отвергать идею, не попытавшись посмотреть на неё с разных сторон. Если что-то в теории может открыть новые возможности, это стоит пробовать.
4. Люди с Win 7 и старым дизайном ВК на вряд ли вложили хоть копейку в ДБГ, это глупое заключение, только что-то мне подсказывает об этом. Эту часть лучше не брать за аргумент. Расширения с дизлайками всё, не работает, YT закрыл API, так что все данные устарели. <br><br>Вопрос, нужно ли держаться тех самых "старичков"? Если они организовывают игру для других, поддерживают проект, не обязательно финансово и в принципе их можно назвать душой компании, тогда определённо да! А если это просто люди с большим таймером в ТАБе, тогда толку никакого. И опять же, это не дело одного дня, всё надо продумать, на каких условиях, как именно, оценить надобность идеи как таковой, провести грамотный опрос и только после возможно что-то и делать.

Не совсем понимаю идею с залогом. Скорее так, донат создан не для возврата. Принцип не поменялся – страх выкинуть деньги на воздух. Но тоже, как мини дополнение к твоему, если он не попадает обратно в бан в течении н-ного времени, потраченная сумма полностью или частично конвертируется в фишки. По сути тоже самое, но деньги остаются в рамках сервера, в чём и была цель.

**Reply by KAGIRA-777** [reply]
1. Я не знаю насколько ты часто играешь на сервере, но игроков которые перманентно нарушают правила достаточно. Да, некоторые из них сейчас в бесконечном бане находятся, но и сейчас есть неплохая часть игроков, которая живет от бана до бана, для них эта сумма в 250-350 незначительна, поверь мне. Сумма в 450-700 рублей была бы нормальной, если еще знать факт того, что она потом конвертируется в фишки. Игроки которые по случайности, либо по неосторожности получили бан будут в любом случаи в плюсе, а вот игроки которые бесконечно нарушают будут в страхе потере денег и их это на некоторое время остановит от бесконечного нарушения правил.
2. Я по сути за это предложение. Я просто сейчас критикую ее т.к в изначальной идеи есть недочеты, я пытаюсь вам открыть глаза на них, чтоб потом не было всяких казусов.
3. Конечно очень странно, что мы сейчас на серьезе обсуждаем Виндовс 7 и старый дизайн ВК, ну и дизлайки на ютуб. Вообще тоже не стоит эту аудиторию недооценивать т.к вы сейчас просто можете откинуть игроков, которые могу в теории задонатить.

Старички по большей части именно ролевики, они умеют отыгрывать и всякие подобные вещи. Вспомни старые организации которые были на ДБГ, в них состояли только старички, которые грамотно могли отыгрывать РП ситуации. Нынче таких организаций нет(почти), все новые игроки отыгрывают всякие шайки, гетто и подобные. Я не против подобного стиля игры, я против того, как они это отыгрывать, а отыгрывают они это ужасно. Сервер нуждается в старичках т.к именно они показывают новым игрокам как отыгрывать РП.

На самом деле надеюсь на то, что мы пришли хоть частично к пониманию ситуации и я вам хоть как-то помог с вашей идеей. Она неплохая, ей просто нужна маленькая доработка.

**Reply by quillincn** [reply]
1. Пожалуйста, не игнорируй факт кулдауна и ценность денег, уже несколько раз писал об этом.
2. На самом деле спасибо за раскрытие темы, в любом случае мне было просто интересно предложить это.
3. Это был не аргумент, просто наитие.

Спасибо за обсуждение! Надеюсь оно как-то поможет другим участникам по другому отнестись к задумке. А дальше дело за ребятами из команды, если понравится, возможно что-то реализуется, а если нет, ничего страшного.

**Comment by quillincn** [comment]
Как дополнение к предложению, в рассечёт так же можно брать следующие факторы, которые могут определять возможность покупки:

- Время игры на сервере
- Количество кармы
- Количество предыдущих банов

Список НЕ окончательный и его МОЖНО дополнять, главное пробовать ДУМАТЬ.
